cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -28,9 +31,9 @@ def move(src, dest):
         shutil.move(src, dest)
 
 
+# output location for python-template cookiecutter
```

Comment:
Is `python-template` wrong?  Maybe just drop it?
```
cookiecutter output location
```

Reply:
the python-template in this comment is correct

Reply:
the variable below is the name of dir where python-template will place its output.